### ISSUE TITLE
Loaded plugins arent "loaded"

### ISSUE THREAD

**Issue by JHarris12345** [r-clancy/PlugMan#81]
I install a new jew and do /plugman load <name>.

It works, the plugin is now in game because it gives me plugin messages. However if I try and execute any commands from that plugin, it doesnt work. 

Also is /plugman unload meant to delete the jar from the server? Because it doesn't, unless it deletes after next restart

**Comment by kennyrkun** [comment]
Answer to the second question: `/plugman unload` is not meant to delete the jar from the server, only unload it. The plugin will be loaded again next time the server is started.

**Comment by JHarris12345** [comment]
Does /plugman disable stop it loading on next restart?

**Comment by kennyrkun** [comment]
No, `/plugman disable` does not stop the plugin from loading when the server is started, it simple disables the plugin during server runtime. This is different from `/plugman unload` which both disables and then unloads the plugin. When the plugin is disabled, it is still loaded on the server, but the server just doesn't let it do anything.

**Comment by JHarris12345** [comment]
So /plugman unload stops the server enabling it on next restart? Different from what the first guy said. Also what about plugman load not having commands for a plugin despite it being installed

**Comment by kennyrkun** [comment]
`/plugman unload ` both disables and unloads the plugin from the server while it is running. the command does not stop the server from reloading the file the next time it starts. this plugin has no mechanism in place to provide that kind of functionality, all loads/unloads/enables/disables happen during runtime and will not persist across server restarts. 

as for plugins not having commands after being loaded using `/plugman load`, I don't know the cause. I suspect the reason is that your plugin is not being loaded correctly and the commands aren't being registered. however, I cannot confirm this without a specific log file from the server.

**Comment by JHarris12345** [comment]
So what is the difference between /plugin disable and /plugin unload

**Comment by kennyrkun** [comment]
The server is not aware of the existence of an unloaded plugin -- it has no idea it's even real. The server *is* aware of disabled plugins, it just ignores them.

For example, plugins that attempt to load at server startup but fail will be disabled by the server. They're still loaded, but the server does not attempt to use them. 

If a plugin is not loaded, the server doesn't even touch it's .jar file or named folder.
